mtags/src/main/scala-3/scala/meta/internal/pc/Params.scala

```diff
@@ -0,0 +1,27 @@
+package scala.meta.internal.pc
+
+import dotty.tools.dotc.core.Contexts.Context
+import dotty.tools.dotc.core.Symbols.Symbol
+import dotty.tools.dotc.core.Flags._
+// maybe we can use Params.scala defined in scala2
+
+case class Params(
```

Comment:
something like https://github.com/scalameta/metals/blob/5345a236983e5fcbaa1d1cae7e03b7590979ab3d/mtags/src/main/scala-2/scala/meta/internal/pc/Params.scala

Reply:
Since I added `Using` kind to Params (in scala3) which shouldn't exist in scala2, it might be reasonable to define two different `Params` classes for scala2 and scala3.